components/brave_ads/core/browser/search_result_ad/search_result_ad_converting_util.cc

```diff
@@ -13,7 +14,7 @@
 #include "base/ranges/algorithm.h"
 #include "base/strings/string_number_conversions.h"
 #include "brave/vendor/bat-native-ads/include/bat/ads/public/interfaces/ads.mojom.h"
-#include "third_party/abseil-cpp/absl/types/optional.h"
+#include "components/schema_org/common/metadata.mojom.h"
 #include "third_party/blink/public/mojom/document_metadata/document_metadata.mojom.h"
```

Comment:
after this migration, `core` cannot depend on `blink` anymore. All `blink` and `content` stuff should be left in `brave_ads/content` target.

I assume you can use `components/schema_org/common` to interact with `array<schema_org.mojom.Entity> entities` directly, skipping the blink-specific `WebPage` structure in `brave_ads/core`.

Reply:
done, removed `WebPage` dep from `core`